packages/taucmdr/cli/commands/trial/tests/test_create.py

```diff
@@ -137,6 +137,30 @@ def test_heap_usage_memory_alloc_profile(self):
         self.assertInLastTrialData("compute")
         self.assertInLastTrialData("malloc")
 
+    def sample_resolution_helper(self, option):
+        self.reset_project_storage()
+        stdout, stderr = self.assertCommandReturnValue(0, measurement_edit_cmd,
+                                            ['sample', '--sample-resolution', option])
+        self.assertIn("Updated measurement 'sample'", stdout)
+        self.assertFalse(stderr)
+        stdout, stderr = self.assertCommandReturnValue(0, select_cmd, ['sample'])
+        self.assertIn("Selected experiment 'targ1-app1-sample'", stdout)
+        self.assertFalse(stderr)
+        self.assertManagedBuild(0, CC, [], 'matmult.c')
+        stdout, stderr = self.assertCommandReturnValue(0, trial_create_cmd, ['./a.out'])
+        self.assertIn("Trial 0 produced 1 profile files", stdout)
+        self.assertIn("TAU_EBS_RESOLUTION="+option, stdout)
+        self.assertFalse(stderr)
+
+    def test_sample_resolution_file(self):
+        self.sample_resolution_helper('file')
```

Comment:
I cannot figure out why the tests for #372 are now failing for me on my python3 branch. It thinks `TAU_EBS_RESOLUTION=line` for all three test cases. I'm wondering if some command output caching is happening somewhere for the python3 implementation that wasn't happening for the python2 implementation?

Reply:
Line is the default if nothing is set. Is the setting of the option not working, or is the environment variable not getting picked up by TAU somehow?